### PR TITLE
Making "#!" bold in hashbang.html

### PR DESCRIPTION
**Pull request by Zelaf**
Made the text Bold in (hopefully) all browsers since it was normal in Firefox.

### CODE REVIEW

**Comment by stefan2904** [comment]
(someone needs to re-sign before tests are able to run through.)

**Comment by ChickenNuggers** [comment]
@stefan2904 please do not approve changes if there is an issue, which as you said, there is one if we need to re-sign the file.

**Comment by KellerFuchs** [comment]
@ChickenNuggers I don't see the issue with that, given that the CI prevents pushing without having updated the signed data.

@Zelaf What's the magic constant 700 in there?

File: src/hashbang.html [hunk 1]
```diff
@@ -54,11 +54,11 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
     <meta name="HandheldFriendly" content="true"/>
     <meta name="MobileOptimized" content="320"/>
-    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
-    <link href='/assets/local.css' rel='stylesheet' type='text/css'>
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:700" rel="stylesheet" type="text/css">
```
Comment (KellerFuchs) [inline src/hashbang.html]: What's the magic constant 700 in there?

File: src/hashbang.html [hunk 2]
```diff
@@ -54,11 +54,11 @@
...
     <meta name="HandheldFriendly" content="true"/>
     <meta name="MobileOptimized" content="320"/>
-    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
-    <link href='/assets/local.css' rel='stylesheet' type='text/css'>
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:700" rel="stylesheet" type="text/css"> 
+    <link href="/assets/local.css" rel="stylesheet" type="text/css">
   </head>
   <body>
-    <script src='/assets/icon.js'></script>
+    <script src="/assets/icon.js"></script>
```
Comment (KellerFuchs) [inline src/hashbang.html]: Why are you replacing `'` with `"` all over the place?

Comment (lrvick) [inline src/hashbang.html]: " is used everywhere else. He is just being consistant

Comment (Zelaf) [inline src/hashbang.html]: Afaik, I'm unsure there is a difference.

File: src/hashbang.html (hunk 1 above)
Comment (KellerFuchs) [inline src/hashbang.html]: Why is it being used literally nowhere: neither in CSS or HTML, not as 700, `bold` or `<b></b>` ?

File: src/hashbang.html (hunk 2 above)
Comment (KellerFuchs) [inline src/hashbang.html]: @Zelaf There is none there, but I can buy being consistent.

File: src/hashbang.html (hunk 1 above)
Comment (alangshall) [inline src/hashbang.html]: Why not use `font-weight: bold;` in local.css? 

I'm also not entirely certain this PR does what it says on the tin. I tried the relevant html locally and saw no cosmetic difference between the modified code and what's in production in either Chrome or Firefox. 

I would find defining what the look of the #! site should be extremely helpful.

**Comment by KellerFuchs** [comment]
I didn't want to approve, I only wanted to remove the “changes required” status...

File: src/local.css [hunk 3]
```diff
@@ -25,7 +25,7 @@ body {
 }
 h1 {
   text-indent:0px;
-  font-family: 'Montserrat', cursive;
+  font-family: 'Montserrat', bold;
```
Comment (KellerFuchs) [inline src/local.css]: As previously commented, bold is AFAIK not a font family.
What is making the whole thing bold here?

**Comment by KellerFuchs** [comment]
Well, that's “fixed”.  For some values of “fixed”.

**Comment by KellerFuchs** [comment]
@alangshall @stefan2904 Please review my changes.

**Comment by KellerFuchs** [comment]
**Boldly** deployed to production

**Pull request merged**